backend/matching-service/spec/matching-queue.spec.js

```diff
@@ -0,0 +1,75 @@
+const MatchingQueue = require('../models/matching-queue');
+const Match = require('../models/match.object');
+
+describe('MatchingQueue', () => {
+  let queue;
+
+  beforeEach(() => {
+    queue = new MatchingQueue();
+  });
+
+  describe('#enqueue', () => {
+    it('adds a match to the queue', () => {
+      const match = new Match("User", "Hard", "Python", null);
+      queue.enqueue(match);
+      expect(queue.getQueueLen()).toEqual(1);
+    });
+
+    it('returns undefined if the user already exists in the queue', () => {
```

Comment:
Should we standardise the message here? As i recall, the jasmine standard was:

`describe({something})`

`it(`should {do something}')`

Reply:
Yup, changed!